cylc/flow/suite_status.py

```diff
@@ -60,7 +60,7 @@ class StopMode(Enum):
     """Suite has reached a state where it can automatically stop"""
 
     AUTO_ON_TASK_FAILURE = 'AUTOMATIC(ON-TASK-FAILURE)'
-    """A task has failed and ``abort if any task fails = True``."""
+    """A task has failed and ``--abort-if-any-task-fails`` was given."""
```

Comment:
```suggestion
   """A task has failed and ``--abort-if-any-task-fails`` was used."""
```
Really just a suggestion. Ignore if you don't like..